ephios/api/templates/oauth2_provider/application_list.html

```diff
@@ -33,6 +33,13 @@ <h4>{% translate "OAuth2 applications" %}</h4>
                         <td>{{ application.created }}</td>
                         <td class="d-flex">
                             <div>
+                                {% if application.federatedhost_set.exists %}
+                                <a class="btn btn-secondary"
+                                   href="{% url "federation:settings" %}">
+                                    <span class="fa fa-cog"></span>
+                                    <span class="d-none d-lg-inline">{% translate "Manage federation" %}</span>
```

Comment:
I think we should add `(experimental)` labels to buttons controlling these features.